plugins/woocommerce/tests/php/src/Internal/Logging/OrderLogsDeletionProcessorTest.php

```
@@ -0,0 +1,266 @@
+		$data_store = new class() extends \WC_Order_Data_Store_CPT {};
+		add_filter( 'woocommerce_order_data_store', fn()=>$data_store, 99999, 0 );
+
+		$this->setup_hpos_and_reset_container( false );
+
+		$actual_function_name = null;
+		$actual_message       = null;
+
+		// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter.FoundAfterLastUsed
+		$this->register_legacy_proxy_function_mocks(
+			array(
+				'wc_doing_it_wrong' => function ( $function_name, $message, $version ) use ( &$actual_function_name, &$actual_message ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$version'. (undefined)

(UnusedFormalParameter)